src/gen_context.c

```diff
@@ -4,9 +4,10 @@
  * file COPYING or http://www.opensource.org/licenses/mit-license.php.*
  **********************************************************************/
 
+#include "libsecp256k1-config.h" // for ECMULT_GEN_PREC_BITS
```

Comment:
It's not crucial but we should make sure that this warning disappears:
```/configure --disable-openssl-tests --with-ecmult-gen-precision=8 --enable-experimental --enable-module-recovery --enable-module-ecdh --with-ecmult-window=22  CC=clang```

```
In file included from src/gen_context.c:9:
src/basic-config.h:33: warning: "ECMULT_WINDOW_SIZE" redefined
   33 | #define ECMULT_WINDOW_SIZE 15
      |
In file included from src/gen_context.c:7:
src/libsecp256k1-config.h:18: note: this is the location of the previous definition
   18 | #define ECMULT_WINDOW_SIZE 22
```

I guess other compilers warn, too.

In the end it does not matter what `ECMULT_WINDOW_SIZE` is for use in `gen_context.c`. I think sticking to 15 is reasonable (but the fact that we ignore the config setting here should be more prominent in the source).

Reply:
> In the end it does not matter what `ECMULT_WINDOW_SIZE` is for use in `gen_context.c`. I think sticking to 15 is reasonable (but the fact that we ignore the config setting here should be more prominent in the source).

Actually it does not matter too much because `gen_context` does not use `ECMULT_WINDOW_SIZE` at all.

Reply:
Fixed via `#undef ECMULT_WINDOW_SIZE` in basic-config.h, like with the other definitions.

Reply:
@real-or-random  sorry to bug, seems ok now? 😊

Reply:
Ping @real-or-random - seems we are really close now :P